### ISSUE TITLE
More details Documentation ?

### ISSUE THREAD

**Issue by ghost** [furnishup/blueprint3d#37]
Hi, anyone here know where can I get a detail documentation ?
I plan to
1) modify the canvas view , I do not know how to start .
2) add more custom 3D object, 
3) add more function like change the color of 3D objects .

Thanks

**Comment by aalavandhaann** [comment]
I agree there should be a documentation. But I can perhaps throw some light. Try to export your model from Blender or any 3D as js object. Supply a thumbnail for the same. To add it to your gallery of items edit the items.js file inside the "examples/js" folder. Add your entry and then you have your item listed. 

Let yourself known if you have any issues to add an item. 

Regards,

#0K

**Comment by nikhildodeja** [comment]
First of all thanks to you: 
It's a really cool tool.
hi I want to do more programming but from where to start i dont know 

 I understand the code... 

My basic requirement is to draw a simple floor and load object or mesh.js from items models.

It's totally written in javascript and typescript. You are converting your typescript into js using grunt.

But heavy object is not render when i convert it into .js file.

Can you have any idea to render object with mesh I have read document on three js site and found some solution but Is it combine with your code or not .

My Requirement is : 
I want to draw floor simple floor without a wall. I want to add object and just move and rotate them on floor. I have just floor type of object like sofa furniture 

And one more thing you are really good OOP's developer.

**Comment by micreales** [comment]
Hi, do you have any idea how can I prevent the rug to penetrate or stick againts the wall when it's rotating ? Thanks, or in short, can we disable the item can penetrate to the wall ?
